### DISCUSSION TITLE
[v2][auto-prompt] Redirect output to file with auto-prompt?

### DISCUSSION THREAD

**Discussion by weibeld** [aws/aws-cli#7569] [Q&A] [answered]
Currently, writing something like this in auto-prompt mode results in an error:

```bash
> aws ec2 describe-instances >out.json
```

The `>out.json` output redirection is interpreted by the AWS CLI rather than by the shell.

Is there any way to redirect the output to a file with auto-prompt?

**Comment by tim-finnigan** [comment, marked as answer]
Hi @weibeld, thanks for reaching out. I'm not aware of a way to currently redirect the output. I brought this up with the team, and they think this should be possible to address as a feature request. I created an issue from this discussion for tracking going forward: https://github.com/aws/aws-cli/issues/7576
